examples/us-dams/README.md

```diff
@@ -0,0 +1,53 @@
+# US dams
+
+This is an [Observable Framework](https://observablehq.com/framework) project. To start the local preview server, run:
+
+```
+npm run dev
+```
+
+Then visit <http://localhost:3000> to preview your project.
+
+For more, see <https://observablehq.com/framework/getting-started>.
+
+## Project structure
+
+A typical Framework project looks like this:
+
+```ini
+.
+├─ docs
```

Comment:
we should remove this part (however if we kept it we should use src instead of docs)